src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -3833,3 +4296,46 @@ def test_read_only_fields_hex(tmpdir):
         "        CALL f2_proxy(3)%halo_exchange(depth=1)\n"
         "      END IF\n")
     assert expected in generated_code
+
+
+def test_mixed_precision_args():
```

Comment:
The answer is yes it can be checked but no it does not work. Our copy of the LFRic infrastructure needs to be updated to include `r_solver_operator_type` and `r_solver_operator_proxy_type`.

Reply:
I've just checked the latest version of lfric_trunk and it does not contain r_solver_operator_type. This functionality must still be on a branch. I have created a new issue #1638, have made the associated test xfail and have referenced the new issue.